apps/studio/src/components/sidebar/core/table_list/StatelessTableListItem.vue

```diff
@@ -0,0 +1,176 @@
+<template>
```

Comment:
Question for my understanding - what makes these 'stateless' vs the original components?

Could you add a nice comment to this file outlining why we need a 'stateless' table list item, maybe linking to the virtual scroll library and such?

Reply:
Will do! :)

Reply:
The original components have states that vanish when it's out of sight or "unmounted" from the virtual list. These states are `expanded`, `pinned`, `draggable` (though I'm not sure we still use draggable?) .

Reply:
Got it. That's some really useful information! Please def add as a comment